advanced-integration/server.js

```diff
@@ -1,20 +1,44 @@
 import "dotenv/config";
 import express from "express";
+import bodyParser from "body-parser";
 import * as paypal from "./paypal-api.js";
-const {PORT = 8888} = process.env;
+
+const { PORT = 8888 } = process.env;
+
+const handleError = (res, error) => {
+  console.error(error);
+  res.status(error.status || 500).send(error.message);
+};
 
 const app = express();
 app.set("view engine", "ejs");
 app.use(express.static("public"));
+app.use(bodyParser.json());
 
-// render checkout page with client id & unique client token
 app.get("/", async (req, res) => {
   const clientId = process.env.CLIENT_ID;
   try {
-    const clientToken = await paypal.generateClientToken();
-    res.render("checkout", { clientId, clientToken });
+    res.render("index", { clientId });
+  } catch (err) {
+    handleError(res, err);
```

Comment:
Will `res.render` ever throw?

Reply:
its possible. I think that try/catch was there for `paypa.generateClientToken` but its possible there could be an issue rendering `index.ejs` or finding the file if the filesystem changes